src/libraries/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -44,6 +44,7 @@
     <Compile Include="$(CoreLibSharedDir)Interop\Windows\Kernel32\Interop.FindClose.cs" Link="Interop\Windows\Interop.FindClose.cs" />
     <Compile Include="$(CoreLibSharedDir)Interop\Windows\Kernel32\Interop.FindFirstFileEx.cs" Link="Interop\Windows\Interop.FindFirstFileEx.cs" />
     <Compile Include="$(CoreLibSharedDir)Interop\Windows\Kernel32\Interop.FormatMessage.cs" Link="Interop\Windows\Interop.FormatMessage.cs" />
+    <Compile Include="$(CoreLibSharedDir)Interop\Windows\Kernel32\Interop.GenericOperations.cs" Link="Interop\Windows\Interop.GenericOperations.cs" />
     <Compile Include="$(CoreLibSharedDir)Interop\Windows\Kernel32\Interop.GET_FILEEX_INFO_LEVELS.cs" Link="Common\Interop\Windows\Interop.GET_FILEEX_INFO_LEVELS.cs" />
     <Compile Include="$(CoreLibSharedDir)Interop\Windows\Kernel32\Interop.GetFileAttributesEx.cs" Link="Common\Interop\Windows\Interop.GetFileAttributesEx.cs" />
     <Compile Include="$(CoreLibSharedDir)Interop\Windows\Kernel32\Interop.GetFullPathNameW.cs" Link="Interop\Windows\Kernel32\Interop.GetFullPathNameW.cs" />
```

Comment:
I'm not too familiar with the csproj files, I'm wondering why in the `Link` some start with `Common` ? Shouldn't they be just Interop directly or I'm missing something ?
My additions of the `Compile` elements are all without `Common` at start of the `Link`.

Reply:
I think what you did was correct.

The Include path variable `$(CoreLibSharedDir)` is now being used in place of `$(CommonPath)`. The name gives more clarity: You should place files shared by both `libraries` and by `coreclr` inside this folder.

Now, when you open a csproj, it is my understanding that the user shouldn't care if the file is shared by coreclr or not, so it's ok to remove the Common prefix from the Link. This means that the other includes should be fixed, not yours. But you don't need to do that change, we can do this separately. 

I'll double check this information with my team, just in case.

Reply:
We haven't been consistent on what the `Link` starts with, unfortunately. I think we will need to get to a consensus of what we want. Basically `Link` will affect how the file structure is displayed on VS, so if it starts with Common it will show it under a `Common` directory. 

I think that's the pattern we wanted to follow in order to know from the editor which files come from `Common` paths. 

@stephentoub thoughts?